sql/database.sql

```diff
@@ -6562,12 +6562,30 @@ CREATE TABLE `lists` (
   `severity_al` VARCHAR( 50 ) DEFAULT NULL,
   `external_id` VARCHAR(20) DEFAULT NULL,
   `list_option_id` VARCHAR(100) DEFAULT NULL COMMENT 'Reference to list_options table',
+  `drug_dosage_instructions` longtext DEFAULT NULL COMMENT 'Medication list dosage instructions',
   PRIMARY KEY  (`id`),
   KEY `pid` (`pid`),
   KEY `type` (`type`),
   UNIQUE KEY `uuid` (`uuid`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1;
 
+
+-- --------------------------------------------------------
+DROP TABLE IF EXISTS `lists_medication`;
+CREATE TABLE `lists_medication` (
+                                    `id` BIGINT(20) NOT NULL AUTO_INCREMENT
+    , `list_id` BIGINT(20) NULL COMMENT 'FK Reference to lists.id'
+    , `drug_dosage_instructions` LONGTEXT NULL COMMENT 'Free text dosage instructions for taking the drug'
```

Comment:
make text fields bare (ok for the COMMENT, though); just trying to keep the text fields consistent since have spent resources in past standardizing these.